package.json

```diff
@@ -55,25 +55,15 @@
     "string.prototype.matchall": "^4.0.6"
   },
   "scripts": {
-    "build-phy": "webpack --env prod --config config/webpack.config.physics.js",
     "build-cs": "webpack --env prod --config config/webpack.config.cs.js",
```

Comment:
Since you've removed the `cs` suffix from some places, you may want to remove it from the name of this script. And also from the webpack config file. I've also seen it in the renderer script/webpack config, as well as jest config.

Reply:
I did initially change these, then decided to revert, as we aren't editing the build / docker files right now so I thought this would have an impact, and we wanted each ticket to be independent.